homeassistant/components/zwave/__init__.py

```diff
@@ -399,11 +411,20 @@ def value_added(node, value):
                 value.enable_poll(polling_intensity)
             else:
                 value.disable_poll()
+            platform = get_platform(component, DOMAIN)
+            device = platform.get_device(
+                node=node, value=value, node_config=node_config, hass=hass)
```

Comment:
We can move `node_config` out of `hass.data` since we pass it in here.

I am not a big fan of using `**kwargs` for get_device. I think that we should standardize it.

```
get_config(node, value, node_config):
```

There is no `hass` there because this all about creating the device. Hass will be set on the entity once added to hass.